samples.json

```diff
@@ -1,27 +1,32 @@
 [
   {
-    "Title": "Python",
-    "Description": "Sample Sds client written in Python language",
-    "Url": "https://github.com/osisoft/sample-ocs-waveform-python"
+    "Title": "Authentication",
+    "Description": "Authenticating to OCS using all client types in various languages",
+    "Url": "https://github.com/osisoft/OSI-Samples-OCS/docs/AUTHENTICATION_README.md"
   },
   {
-    "Title": ".NET",
-    "Description": "Sample Sds client written in C#",
-    "Url": "https://github.com/osisoft/OSI-Samples-OCS/tree/main/docs/SDS_WAVEFORM_DOTNET_README.md"
+    "Title": "SDS",
+    "Description": "SDS common actions written in various languages",
+    "Url": "https://github.com/osisoft/OSI-Samples-OCS/docs/COMMON_ACTIONS.md"
```

Comment:
just making this comment as a reminder placeholder thing, in case you do end up adding _README to the end of these file names, to change them here too.

Reply:
I don't like the _readme, if people think that we should do this, than we can